<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsTextMessageInlineComponent</class>
 <widget class="QFrame" name="CSettingsTextMessageInlineComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="hl_TextMessageSettings">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_Overlay">
     <property name="toolTip">
      <string>Overlay</string>
     </property>
     <property name="text">
      <string>Ovl.:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_Private">
     <property name="text">
      <string>private</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_Supervisor">
     <property name="toolTip">
      <string>supervisor</string>
     </property>
     <property name="text">
      <string>super.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_Selcal">
     <property name="text">
      <string>SELCAL</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_Frequency">
     <property name="toolTip">
      <string>frequency</string>
     </property>
     <property name="text">
      <string>freq.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_All">
     <property name="text">
      <string>all</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
